assets/xml/objects/object_dinofos.xml

```diff
@@ -1,71 +1,76 @@
+
+        <DList Name="gDinolfosKnifeDL" Offset="0x5B10" />
+        <DList Name="gDinolfosLowerTailDL" Offset="0x5D00" />
+        <DList Name="gDinolfosUpperTailDL" Offset="0x5E60" />
+        <DList Name="gDinolfosUpperBodyDL" Offset="0x5FC8" />
+        <DList Name="gDinolfosNeckDL" Offset="0x61F8" />
+        <DList Name="gDinolfosHeadDL" Offset="0x6440" />
+        <DList Name="gDinolfosJawDL" Offset="0x6930" />
+        <DList Name="gDinolfosRightUpperArmDL" Offset="0x6C28" />
+        <DList Name="gDinolfosRightForearmDL" Offset="0x6D68" />
+        <DList Name="gDinolfosRightHandDL" Offset="0x6E58" />
+        <DList Name="gDinolfosLeftUpperArmDL" Offset="0x70A0" />
+        <DList Name="gDinolfosLeftForearmDL" Offset="0x71E0" />
+        <DList Name="gDinolfosLeftHandDL" Offset="0x72D0" />
+        <DList Name="gDinolfosLowerBodyDL" Offset="0x7450" />
+        <DList Name="gDinolfosRightUpperLegDL" Offset="0x7690" />
+        <DList Name="gDinolfosRightLowerLegDL" Offset="0x78E0" />
+        <DList Name="gDinolfosRightClawsDL" Offset="0x7AC8" />
+        <DList Name="gDinolfosLeftUpperLegDL" Offset="0x7C20" />
+        <DList Name="gDinolfosLeftLowerLegDL" Offset="0x7E70" />
+        <DList Name="gDinolfosLeftClawsDL" Offset="0x8058" />
+
+        <Texture Name="gDinolfosTLUT" OutName="dinolfos_tlut" Format="rgba16" Width="16" Height="16" Offset="0x81B0" />
+        <Texture Name="gDinolfosSkinTex" OutName="dinolfos_skin" Format="ci8" Width="16" Height="16" Offset="0x83B0" />
+        <Texture Name="gDinolfosScaleTex" OutName="dinolfos_scale" Format="rgba16" Width="16" Height="16" Offset="0x84B0" />
+        <Texture Name="gDinolfosArmorTex" OutName="dinolfos_armor" Format="rgba16" Width="16" Height="16" Offset="0x86B0" />
+        <Texture Name="gDinolfosSpikeTex" OutName="dinolfos_spike" Format="rgba16" Width="8" Height="8" Offset="0x88B0" />
+        <Texture Name="gDinolfosInnerMouth" OutName="dinolfos_inner_mouth" Format="rgba16" Width="16" Height="16" Offset="0x8930" />
+        <Texture Name="gDinolfosKnifeHole" OutName="dinolfos_knife_hole" Format="rgba16" Width="8" Height="8" Offset="0x8B30" />
+        <Texture Name="gDinolfosJawTex" OutName="dinolfos_jaw" Format="rgba16" Width="8" Height="16" Offset="0x8BB0" />
+        <Texture Name="gDinolfosEyeHoleTex" OutName="dinolfos_eye_hole" Format="rgba16" Width="16" Height="8" Offset="0x8CB0" />
+        <Texture Name="gDinolfosClawTex" OutName="dinolfos_claw" Format="rgba16" Width="8" Height="8" Offset="0x8DB0" />
+        <Texture Name="gDinolfosEyeOpenTex" OutName="dinolfos_eye_open" Format="rgba16" Width="16" Height="16" Offset="0x8E30" />
+        <Texture Name="gDinolfosEyeHalfTex" OutName="dinolfos_eye_half" Format="rgba16" Width="16" Height="16" Offset="0x9030" />
+        <Texture Name="gDinolfosEyeClosedTex" OutName="dinolfos_eye_closed" Format="rgba16" Width="16" Height="16" Offset="0x9230" />
```

Comment:
Not sure if you have to change it, but normally, because the objects get their own folders which separates them, we don't have to add the object or actor name to the texture output file names, such that "dinolfos_skin" would just be "skin", etc.

Reply:
I've been putting a prefix on everything (like what has been done here) because there is no guarantee that these will all be in different folders going forward. Plus, it means people can put all the textures in one folder if they want to, and there won't be any conflicts.